table/block_based_table_reader.cc

```diff
@@ -2007,7 +2007,8 @@ void BlockBasedTableIterator::FindKeyForward() {
     }
 
     auto ukey = user_key();
-    if (icomp_.user_comparator()->Compare(ukey, *range_tombstone_end_) >= 0) {
+    if (icomp_.user_comparator()->Compare(ukey, *range_tombstone_.end_key()) >=
+        0) {
```

Comment:
This formatting is particularly egregious, but I don't think there's anything to be done about it.